backend/app/services/adapters/executor_kinds.py

```
@@ -540,10 +556,53 @@ def _update_task_status_based_on_subtasks(self, db: Session, task_id: int) -> No
+            async def delete_executors_async():
+                """Asynchronously delete all executors for the task"""
+                for executor in executors_data:
+                    try:
+                        logger.info(f"Auto deleting executor for task {task_id}: ns={executor['namespace']} name={executor['name']}")
+                        result = await self.delete_executor_task_async(
+                            executor['name'],
+                            executor['namespace']
+                        )
+                        logger.info(f"Successfully auto deleted executor: {result}")
+                        
+                    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)